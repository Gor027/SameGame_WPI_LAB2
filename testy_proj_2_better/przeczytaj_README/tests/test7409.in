  pp##p#  ## p# #p#p p #   p##p## p##pp pp #p   p##p  pppp p## ##### 
 #  pp #p pp# p#p  ### ### ##   p p#p p pp#ppp   pp#p
 pp  #pp##pp p#p#  p##  p ##pppp  ###p #p#p# pp###
##
#    #ppp#  p####p#p   #  p##p# 
p   ##### p
  ## p pp ##p## #   p p #p
####  # p   pp#ppppppp## #p
  ## ##ppp##p#  ### ppp  p  p
 p      #pppp #pppppp # p#p#p p####pp p # #  ### p ## # pp ## ###p  ##pp#
     p#p ### pppppp pppp#ppppp ##p# ####pp# ##p   p#p #p# #p#  #pp   #  p  p
#  #ppp#   p p p#  p pp ###
#p pp#pp  ppp###ppppp## pp  p#pp p   ##p p#pp ##p p  p p#p#pp   # pp  #pp ## p##
