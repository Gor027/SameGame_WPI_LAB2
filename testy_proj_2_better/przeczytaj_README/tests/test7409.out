                #
     # #        p
     pp#  ## p# #
     #ppp pp pp##  p#p   #  #
     #pp# pp###p#p####p #p  #
     ###p p#p###pp#pp## p##pp
     p#p####pp#p#pppppp##pp###p#  p # p pp#
 #   ##pp##pp##pp##ppp##p#p#pppp# p#ppp ppp#ppp  ##p    p  p#   #
#p  pp#pppp#pp#pppppppp##pp#pp##p####pp#p##ppp##ppp#p pppp ##  ###      #
#pppppp#pp#ppppp#pppppp###p#ppp##p##p####p####p###p## #p#p#p#  #pp## #ppp p
ppppp#pp##ppp###ppppp##ppp#pp#pp#p#p###ppp#pp###ppp#pp#p#p#pp####ppp###pp##p##
